modules/ocf_jenkins/files/autoclean-docker

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Deletes old Docker junk to keep /var/lib/docker at a resonable size.
+set -euo pipefail
+
+fs_usage() {
+    df --output=pcent /var/lib/docker | sed -r 's/^ *([0-9]+)%$/\1/; t; d'
```

Comment:
I think doing something like `df --output=pcent /var/lib/docker | tail -n 1 | sed -r 's/^ *([0-9]+)%$/\1/'` would be clearer than using `t` and `d` from within `sed`, even though it's more commands. I just don't think anyone will understand what those `sed` commands do, whereas `tail` is pretty easy to understand.